src/universal/modules/welcome/components/ProgressDots/ProgressDots.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component, PropTypes } from 'react';
+import look, { StyleSheet } from 'react-look';
+import theme from 'universal/styles/theme';
+
+const combineStyles = StyleSheet.combineStyles;
+
+const onClick = event => {
+  event.preventDefault();
+  console.log('TODO: Navigate to step');
+};
+
+let styles = {};
+
+@look
+// eslint-disable-next-line react/prefer-stateless-function
+export default class ProgressDots extends Component {
+  static propTypes = {
+    // children included here for multi-part landing pages (FAQs, pricing, cha la la)
+    // children: PropTypes.element,
+    numDots: PropTypes.number.isRequired, // how many total dots shall we draw?
+    numCompleted: PropTypes.number,       // how many of the dots are completed?
+    currentDot: PropTypes.number,         // which dot (1=first dot) is the user on now?
+  };
+
+  renderDot(idx) {
+    let { numCompleted, currentDot } = this.props;
+    numCompleted--;
+    currentDot--;
+    let dotStyle = null;
+
+    if (idx === currentDot) {
+      /* we're the active dot */
+      dotStyle = combineStyles(styles.progressDot, styles.progressDotCurrent);
+    } else {
+      if (idx <= numCompleted) {
+        /* render a completed dot */
+        dotStyle = combineStyles(styles.progressDot, styles.progressDotCompleted);
+      } else {
+        /* a dot for the future! */
+        dotStyle = styles.progressDot;
+      }
+    }
+
+    return (
+      <a className={dotStyle} href="#" key={idx} onClick={onClick}>
```

Comment:
Not sure @ackernaut's reason, but my understanding is that all a tags require `href` by spec. Anchor is ~unity operator here.